smart_campus/app/templates/app/login.html

```diff
@@ -0,0 +1,59 @@
+
+{% extends "app/base.html" %}
+{% load static %}
+{% block title %} Smart Campus Login {% endblock %}
+
+
+{% block stylesheet %}{% static "app/css/login.css" %}{% endblock %}
+{% block nav %}{% endblock %}
+{% block content %}
+  <div class="container">  
+            <div class="col-lg-6 col-md-6 col-sm-8  loginbox">
```

Comment:
Please fix all indentation in your HTML files.
As consulting with one familiar with the frontend, two space is the indentation convention of HTML.

Reply:
okay, I'll fix them
Sorry, I did not pay much attention on the HTML pages